x/ibc/03-connection/keeper/handshake.go

```diff
@@ -140,17 +129,16 @@ func (k Keeper) ConnOpenAck(
 	consensusHeight uint64,
 ) error {
 	// XXX: blocked by #5078
-	/*
-		if consensusHeight > uint64(ctx.BlockHeight()) {
-			return sdkerrors.Wrap(ibctypes.ErrInvalidHeight, "invalid consensus height")
-		}
-	*/
+	// if consensusHeight > uint64(ctx.BlockHeight()) {
```

Comment:
What is this blocked by? https://github.com/cosmos/cosmos-sdk/issues/5078 is closed.

Reply:
I thought #5475 was blocked by this PR?